src/components/form-create-account/form-create-account.js

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from "react";
+import { FormInput } from "../form-input/form-input";
+import { Select } from "../select/select";
+import { RadioList } from "../radio-list/radio-list";
+import { PasswordWithConfirm } from "../password-with-confirm/password-with-confirm";
+import formCreateAccountPropTypes from "./form-create-account.propTypes";
+import formCreateAccountDefaultValues from "./form-create-account.defaultProps";
+
+/**
+ *
+ * @param {*} ageOptions
+ * @param {*} genderOptions
+ * @param {*} illnesesOptions
+ * @param {*} lastPeriodInformationOptions
+ */
+export const FormCreateAccount = ({
+  ageOptions,
+  genderOptions,
+  illnesesOptions,
+  lastPeriodInformationOptions,
+  objectChanged,
+  countriesList,
+  citiesList,
+  dataProcessingApprovalText
+}) => {
+  const [fullObject, setFullObject] = useState({});
+  const [cities, setCities] = useState([]);
+
+  const input_OnChange = args => {
+    let value =
+      args.target.type === "checkbox" ? args.target.checked : args.target.value;
+    let changedObj = { ...fullObject, [args.target.name]: value };
+    objectChanged && objectChanged(changedObj);
+    setFullObject(changedObj);
+  };
+
+  const optiuniUltimaPerioada_OnChange = args => {
```

Comment:
Could we please write the code in English? We will want to share it with other countries.

Reply:
Sure, no problem - but hen shouldn't we also localize the input form labels and UI text?

Reply:
That will definitely come later.